mk/dev.mk

```diff
@@ -88,3 +88,8 @@ dev/sync-demo:
 	curl -s --fail https://raw.githubusercontent.com/kumahq/kuma-counter-demo/master/gateway.yaml | \
 		sed 's/\([^/]\)kuma-demo/\1{{ .Namespace }}/g' \
 		> app/kumactl/data/install/k8s/demo/gateway.yaml
+
+CIRCLECI_BADGE ?= "[![CircleCI {{branch}}](https://img.shields.io/circleci/build/github/kumahq/kuma/{{branch}}?label={{branch}})](https://circleci.com/gh/kumahq/kuma/tree/{{branch}})"
+.PHONY: dev/repo-health
+dev/repo-health:
+	go run $(TOOLS_DIR)/dev/repo-health.go -action README -circleci-badge $(CIRCLECI_BADGE)
```

Comment:
nitpick: I'm guessing this could've been done with some `jq` and `sed` but if it's already written in go then no reason to change this.

Reply:
Yeah I started going down the road of jq and sed and then just thought it was easier and quicker to write in go